deploy/contents/install/install-config

```diff
@@ -54,6 +54,13 @@ CP_PREF_TEMPLATES_ERROR_PAGES_DIRECTORY_EXT=/opt/api/ext/templates/error-templat
 CP_PREF_API_STATIC_DIRECTORY_EXT=/opt/api/ext/templates/static
 CP_PREF_API_ETC_DIRECTORY=/opt/api/etc
 
+# cp-gitlab-db config
+CP_GITLAB_PSG_VERSION=9.6
```

Comment:
I think we already have settings for `GITLAB_DATABASE` please check that we don't have dupicates and these values correspond to each other